frontend/src/components/UserSettings/UserSettings.js

```diff
@@ -101,8 +101,7 @@ class UserSettings extends React.Component {
     email: '',
     first_name: '',
     last_name: '',
-    phone: '1',
-    password: '',
```

Comment:
The password is on state for future use when we implement the reset password functionality, it is not there to display the current password.